docs/node/determinism.md

```diff
@@ -24,5 +24,5 @@ Workflow code is bundled on Worker creation using [Webpack](https://webpack.js.o
 - `uuid4` - provided by the runtime
 - `Date` - replaced by the runtime
 - `WeakRef | WeakMap | WeakSet` - can not be used as GC is non-deterministic, deleted by the runtime
-- Timers - `setTimeout` and `clearTimeout` are replaced by the runtime. We recommend you use the `@temporal/workflow` package's exported `sleep` function because it plays well with [cancellation](/docs/node/workflows): `import { sleep } from '@temporalio/workflow'`
+- Timers - `setTimeout` and `clearTimeout` are replaced by the runtime. We recommend you use the `@temporal/workflow` package's exported `sleep` function because it plays well with [cancellation](/docs/node/cancellation-scopes): `import { sleep } from '@temporalio/workflow'`
```

Comment:
```suggestion
- Timers - `setTimeout` and `clearTimeout` are replaced by the runtime. We recommend you use the `@temporal/workflow` package's exported `sleep` function because it plays well with [Node SDK cancellation scopes](/docs/node/cancellation-scopes): `import { sleep } from '@temporalio/workflow'`
```

Reply:
I’d just fix the link, I don’t think we need to mention the node sdk, it should be clear in this context